starter_code/components.js

```diff
@@ -0,0 +1,32 @@
+class AwesomePictureThing extends React.Component {
+    
+    constructor(){
```

Comment:
Don't forget to accept `props` as an argument to the constructor function and then pass it to `super`! Like so:

```js
constructor(props) {
  super(props);
  this.state = { show: false };
}
```